src/test/java/entities/ExporterTest.java

```diff
@@ -91,7 +92,7 @@ public ExporterTest() {
 
     /**
      * Asserts that the two files of the exported inventory exists at the specificed location
-     * @throws IOException as required by Exporter
+     * @throws IOException as required by Exporter class
      */
 
     @Test
```

Comment:
Is it possible to have the user specify the export path instead of hard coding it in?